### PR TITLE
Update Organization Permissions Documentation

### PR DESCRIPTION
**Pull request by juliannatetreault**
### What
Recently, we separated "Project permissions" and "Workspace permissions" in the Organization Permissions UI, allowing for greater flexibility in the future if we decide to add more organization-level permissions, and to keep the UI consistent with both the API and TFE Terraform provider. This updates the relevant documentation, namely, the [TFC Team Permissions page](https://developer.hashicorp.com/terraform/cloud-docs/users-teams-organizations/permissions#organization-permissions).

### Why
As mentioned above, we recently separated the "Project" and "Workspace" permission radio selectors from one another in the Organization Permissions UI for consistency and flexibility. Please see [this PR](https://github.com/hashicorp/atlas/pull/14715) for further context around the changes introduced.

### Links
- [JIRA](https://hashicorp.atlassian.net/browse/TF-4992)
- [RFC](https://hashidocs.hashicorp.services/document/1lRqrBATuF0LcEsRvUEPtwzHH0HMOJMe6pYsTPpUywaI)

#### Preview of the updated [Organization Permissions](https://github.com/hashicorp/terraform-docs-common/blob/fd7e5b71acc93a65cac7940da930f5beceaec135/website/docs/cloud-docs/users-teams-organizations/permissions.mdx#organization-permissions):
Screen Shot 2023-03-21 at 8 20 09 AM

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [x] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [x] Description links to related pull requests or issues, if any.

#### Content
- [x] (N/A) Redirects have been added for moved, renamed, or deleted pages. This requires a separate PR in the [`terraform-website` repository](https://github.com/hashicorp/terraform-website) `redirects.next.js` file.
- [x] API documentation and the API Changelog have been updated. 
- [x] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [x] Pages with related content are updated and link to this content when appropriate.
- [x] (N/A) Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [x] (N/A) New pages have metadata (page name and description) at the top.
- [x] (N/A) New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [x] (N/A) New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [x] (N/A) UI elements (button names, page names, etc.) are bolded.
- [x] The Vercel website preview successfully deployed.

#### Reviews
- [x] I or someone else reviewed the content for technical accuracy.
- [x] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204239611470766

### CODE REVIEW

File: website/docs/cloud-docs/users-teams-organizations/permissions.mdx
Comment (emlanctot) [inline website/docs/cloud-docs/users-teams-organizations/permissions.mdx:184]: Should this be the "None" that is reflected on the setting page? All the other names seem to correspond directly to the name in the UI

Comment (juliannatetreault) [reply website/docs/cloud-docs/users-teams-organizations/permissions.mdx:184]: I initially had this heading as "None" for consistency between the UI and docs but upon rereading it, felt like "None" wasn't a very descriptive heading, even though it's nested under the org permissions section. This being said though, I definitely don't feel strongly about either heading and do like keeping things consistent, so I'm happy to defer to y'all! What do you, @emlanctot? 🤔

Comment (emlanctot) [reply website/docs/cloud-docs/users-teams-organizations/permissions.mdx:184]: I don't feel strongly, I think I would keep it consistent to the UI headers. But I would defer to education team !

File: website/docs/cloud-docs/users-teams-organizations/permissions.mdx
Comment (rkoron007) [inline website/docs/cloud-docs/users-teams-organizations/permissions.mdx:226]:
```suggestion
Members do not have access to projects or workspaces. You can grant permissions to individual projects or workspaces through [Project Permissions](/terraform/cloud-docs/users-teams-organizations/permissions#project-permissions) or [Workspace Permissions](/terraform/cloud-docs/users-teams-organizations/permissions#workspace-permissions).
```

File: website/docs/cloud-docs/users-teams-organizations/permissions.mdx
Comment (rkoron007) [inline website/docs/cloud-docs/users-teams-organizations/permissions.mdx:230]:
```suggestion
Members can view all workspaces within the organization. This lets users:
```

File: website/docs/cloud-docs/users-teams-organizations/permissions.mdx
Comment (rkoron007) [inline website/docs/cloud-docs/users-teams-organizations/permissions.mdx:235]:
```suggestion
Members can create and manage all workspaces within the organization. This lets users:
```

File: website/docs/cloud-docs/users-teams-organizations/permissions.mdx
Comment (rkoron007) [inline website/docs/cloud-docs/users-teams-organizations/permissions.mdx:236]:
```suggestion
- Perform any action that requires admin permissions in those workspaces.
```

File: website/docs/cloud-docs/users-teams-organizations/permissions.mdx
Comment (rkoron007) [inline website/docs/cloud-docs/users-teams-organizations/permissions.mdx:237]:
```suggestion
- Create new workspaces within the organization's **Default Project**, an action that is otherwise only available to organization owners. 
```

File: website/docs/cloud-docs/workspaces/organize-workspaces-with-projects.mdx
Comment (rkoron007) [inline website/docs/cloud-docs/workspaces/organize-workspaces-with-projects.mdx:19]: To mark that these are two separate permissions I'd recommend unbolding the "and" in the middle:
```suggestion
You must be a member of a team with **Manage all projects** and **Manage all workspaces** [organization-level permissions](/terraform/cloud-docs/users-teams-organizations/permissions#organization-permissions) permissions to create a project.
```

File: website/docs/cloud-docs/workspaces/settings/access.mdx
Comment (rkoron007) [inline website/docs/cloud-docs/workspaces/settings/access.mdx:30]: How would feel about restructuring this as a list? Right now it's starting to feel pretty dense, and I think a list would helps folks parse this a bit easier:
```suggestion
When a user creates a workspace, the following teams can access that workspace with full admin permissions:
- [the owners team](/terraform/cloud-docs/users-teams-organizations/teams#the-owners-team)
- teams with "Manage all workspaces" and/or “Manage all projects” [organization permissions](/terraform/cloud-docs/users-teams-organizations/permissions#project-permissions)
- teams with “Project Admin” project permissions
You cannot override these teams' permissions through the workspace's specific permissions.
```

@ritsok for thoughts on style!

**Review by rkoron007 — CHANGES_REQUESTED** [review]
Adding my first pass for style!  (Also, I'm still onboarding so you might want to wait for @ritsok's feedback before implementing any of this 😅)

**Comment by ritsok** [comment]
👍 for all of rose's suggestions. thank you for the thorough review!

**Comment by juliannatetreault** [comment]
> 👍 for all of rose's suggestions. thank you for the thorough review!

Amazing! Thank you both so much, @rkoron007 and @ritsok

File: website/docs/cloud-docs/users-teams-organizations/permissions.mdx
Comment (juliannatetreault) [reply website/docs/cloud-docs/users-teams-organizations/permissions.mdx:215]: Great suggestion!

**Review by rkoron007 — APPROVED** [review]
Looks great to me! ✨

File: website/docs/cloud-docs/api-docs/workspaces.mdx
Comment (rkoron007) [inline website/docs/cloud-docs/api-docs/workspaces.mdx:57]: Actually, should "Manage all projects" be included in this note, too? If that [permission level has all the of permissions of "Manage all workspaces"](https://terraform-docs-common-git-juliannatetreault-up-43c436-hashicorp.vercel.app/terraform/cloud-docs/users-teams-organizations/permissions#manage-all-projects)?

Comment (juliannatetreault) [reply website/docs/cloud-docs/api-docs/workspaces.mdx]: I've updated this note -- let me know what you think, @rkoron007!

Comment (rkoron007) [reply website/docs/cloud-docs/api-docs/workspaces.mdx]: Looks perfect; thank you so much!

**Pull request merged**
